.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+<!--- Please read the following. Pull requests that do not adhere to these guidelines will be closed. -->
+
+<!--- Each pull request must, with the exception of minor documentation fixes, be associated with an open issue.  -->
+<!--- If a corresponding issue does not exist please stop. Instead, create an issue so we can discuss the change first. -->
+
+<!--- If there is an associated open issue, then the next step is to run through the relevant checklist: -->
```

Comment:
> run through the relevant checklist

I'm not sure what you intend the reader to do here. Is it: "make sure you've read the relevant developer guide"?